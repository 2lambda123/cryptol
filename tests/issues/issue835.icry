:m Float
:t (zero : Float 5 11) <$ zero
