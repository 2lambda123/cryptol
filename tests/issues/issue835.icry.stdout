Loading module Cryptol
Loading module Cryptol
Loading module Float

[error] at issue835.icry:2:4--2:31:
  â€¢ Type `Float 5 11` does not support signed comparisons.
      arising from
      use of expression (<$)
      at issue835.icry:2:4--2:31
